integrations/openai/integration.definition.ts

```diff
@@ -11,7 +11,21 @@ export default new IntegrationDefinition({
       title: 'Generate Content',
       description: 'Generate content using any OpenAI model as LLM',
       input: {
-        schema: llm.schemas.GenerateContentInputSchema,
+        schema: llm.schemas.GenerateContentInputSchema.extend({
+          model: z
+            .enum([
+              'gpt-4o',
```

Comment:
We need to remove the short model names from this list (like `gpt-4o`, `gpt-4-turbo`, etc.) as OpenAI can point those short model names to a newer model with different pricing at any time, which will result in billing and maintenance headaches.

The allowed model names are defined in the model cost parameter here: https://github.com/botpress/botpress/blob/master/integrations/openai/src/index.ts#L19

Reply:
Got it ! I updated the list and fixed the types so we ensure that the provided models have a cost entry